requirements.txt

```diff
@@ -1,3 +1,7 @@
 behave~=1.2.6
 #notebook~=7.2.0
 #linkml~=1.7.10
+aioflureedb[domainapi]~=0.8.3
+jupyter~=1.0.0
```

Comment:
let's not depend on this metapackage if not every notebook feature is needed (use the notebook one instead, and we can move these to a separate file as they are optional)

Reply:
Done